plugin/cache/prefetch_test.go

```diff
@@ -18,6 +18,7 @@ func TestPrefetch(t *testing.T) {
 		qname         string
 		ttl           int
 		prefetch      int
+		staleUpTo     time.Duration
```

Comment:
where is this used? I only see assignments to staleUpTo ?

Reply:
used here... https://github.com/coredns/coredns/pull/4367/files#diff-b6eb3644f59cc77177fa546593ae1b637cfb4699ba5b1664df21574eb874e2acR125

Reply:
I see. lgtm